pdb-alignment-web/src/main/java/org/cbioportal/pdb_annotation/web/controllers/MainGetAlignmentsController.java

```diff
@@ -0,0 +1,323 @@
+package org.cbioportal.pdb_annotation.web.controllers;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.log4j.Logger;
+import org.cbioportal.pdb_annotation.scripts.PdbScriptsPipelineRunCommand;
+import org.cbioportal.pdb_annotation.util.ReadConfig;
+import org.cbioportal.pdb_annotation.web.domain.AlignmentRepository;
+import org.cbioportal.pdb_annotation.web.domain.EnsemblRepository;
+import org.cbioportal.pdb_annotation.web.models.Alignment;
+import org.cbioportal.pdb_annotation.web.models.Ensembl;
+import org.cbioportal.pdb_annotation.web.models.InputAlignment;
+import org.cbioportal.pdb_annotation.web.models.InputSequence;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+/**
+ * 
+ * Main controller getAlignments: Get Alignments
+ * 
+ * @author Juexin Wang
+ *
+ */
+@RestController // shorthand for @Controller, @ResponseBody
+@CrossOrigin(origins = "*") // allow all cross-domain requests
+@Api(tags = "Get Alignments", description = "ensembl/uniprot/sequences")
+@RequestMapping(value = "/api/")
+public class MainGetAlignmentsController {
+    final static Logger log = Logger.getLogger(MainGetAlignmentsController.class);
+
+    @Autowired
+    private AlignmentRepository alignmentRepository;
+    @Autowired
+    private EnsemblRepository ensemblRepository;
+    @Autowired
+    private SeqIdAlignmentController seqController;
+
+    @RequestMapping(value = "/alignments/{id_type}/{id:.+}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
these look like the visible controllers in swagger. I guess these are in a different module/package